### ISSUE TITLE
Systematic Frame size error and Crc error when doing Reset

### ISSUE THREAD

**Issue by pipiche38** [sasu-drooz/Domoticz-Zigate#369]
Since 3.0f firmware , getting a systematic Frame and CRC errors when doing a Software RESET of the Zigate.

```
Feb 25 16:45:46 rasp domoticz[5635]: Error: (Zigate-DEV) onMessage : Frame size is bad, computed = 12 received = 32
Feb 25 16:45:46 rasp domoticz[5635]: Error: (Zigate-DEV) onMessage : Frame CRC is bad, computed = 106 received = 148
```

**Comment by pipiche38** [comment]
Feb 25 17:24:48 rasp domoticz[6195]: Error: (Zigate-DEV) onMessage : Frame size is bad, computed = 12 received = 32
Feb 25 17:24:48 rasp domoticz[6195]: Error: (Zigate-DEV) onMessage : Frame CRC is bad, computed = 106 received = 148

  uncoded msg: bytearray(b'\x01\x80\x02\x10\x02\x10\x02\x15\x94\x02\x10\x02\x10\x02\x10\x11\x02\x10\xff\x01\x80$\x02\x10\x02\x1d\xe0\x02\x11\x02\x10\x02\x10\x02\x10\x15\x8d\x02\x10\x02\x11\xed\xed\xde\x02\x1f\x02\x10\x03')
 decoded msg: bytearray(b'\x01\x80\x00\x00\x05\x94\x00\x00\x00\x11\x00\xff\x01\x80$\x00\r\xe0\x01\x00\x00\x00\x15\x8d\x00\x01\xed\xed\xde\x0f\x00\x03')

**Comment by pipiche38** [comment]
ler's see what is happening with ZGUI

**Comment by pipiche38** [comment]
ZGUI

13:44:35.896 -> 01 02 10 11 02 10 02 10 11 03
13:44:36.065 <- 01 80 24 00 0D E0 01 00 00 00 15 8D 00 01 ED ED DE 0F 00 03
13:44:36.147 <- 01 80 06 00 02 85 01 00 03

Looks like ZGUI didn't display 0x8000 !! Does it get also a CRC/FRAME error ?

**Comment by pipiche38** [comment]
Reported to Akila on https://github.com/fairecasoimeme/ZiGate/issues/164

**Comment by pipiche38** [comment]
Seems to be confirmed as a Zigate Bug

**Comment by pipiche38** [comment]
Fixed with firmware 3.1a and Plugin 4.4
